packages/api/src/platforms/vtex/resolvers/region.ts

```diff
@@ -0,0 +1,6 @@
+import type { Resolver } from '..'
+import type { StoreRegion as Root } from '../clients/commerce/types/StoreRegion'
+
+export const Region: Record<string, Resolver<Root>> = {
```

Comment:
I believe this resolver should not be here, only the `query` resolver. Does this make sense, @emersonlaurentino?